include/dsn/tool-api/task.h

```diff
@@ -250,6 +250,8 @@ class task : public ref_counter, public extensible_object<task, 4>, public trans
     static rpc_engine *get_current_rpc();
     static disk_engine *get_current_disk();
     static env_provider *get_current_env();
+    // code is in dsn/dist/replication/replication.codes.h
+    static std::vector<task_worker *> &get_info(int code);
```

Comment:
```suggestion
    static std::vector<task_worker *> &get_task_code_threads_info(task_code code);
```

Don't make it static member in `class task`, put it in global space.